Kinvey-Xamarin/Store/DataStore.cs

```diff
@@ -101,6 +101,9 @@ public void SetCustomRequestProperty (string key, JObject value)
 			this.customRequestProperties.Add (key, value);
 		}
 
+		KinveyRealtimeDelegate<T> RealtimeCallback { get; set; }
```

Comment:
I'll think about this a little more before changing the names.  Part of the issue here is that `delegate` is a keyword.